examples/example.krun

```diff
@@ -73,3 +73,6 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+# Autoload rmsr kernel module (on by defualt)
+# AUTOLOAD_RMSR = True
```

Comment:
I don't have any good intuition on this; might it be possible to be running Krun on a machine that doesn't support msr, or a kernel that doesn't?